pynamodb/settings.py

```diff
@@ -0,0 +1,38 @@
+import imp
+import os
+from os import getenv
+from botocore.vendored import requests
+import constants
+
+import logging
+
+log = logging.getLogger(__name__)
+
+OVERRIDE_SETTINGS_PATH = getenv('PYNAMO_CONFIG', '/etc/pynamodb/settings_override.py')
+
+settings = {}
+
+class RequestSessionWithHeaders(requests.Session):
```

Comment:
i'm not sure why this still exists.

we will create a class with headers and pass it in settings_override.py